extensions/azurecore/src/azureResource/services/terminalService.ts

```diff
@@ -5,13 +5,42 @@
 
 import * as vscode from 'vscode';
 import * as nls from 'vscode-nls';
-import axios, { AxiosRequestConfig } from 'axios';
+import axios, { AxiosRequestConfig, AxiosResponse } from 'axios';
 import * as WS from 'ws';
 
 import { IAzureTerminalService } from '../interfaces';
 import { AzureAccount, AzureAccountSecurityToken, Tenant } from '../../account-provider/interfaces';
 
 const localize = nls.loadMessageBundle();
+
+
+const handleNeverUsed = async (): Promise<void> => {
+	const neverUsedString = localize('azure.coudTerminal.neverUsed', "It seems you have never used Azure Cloud Shell. Please visit https://shell.azure.com/ to get started.");
```

Comment:
Maybe include a bit more saying that after they set up initial configuration they can come back and use the shell in ADS